dev/io.openliberty.wsoc.21.internal_fat/fat/src/io/openliberty/wsoc/common/Constants.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2014 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
```

Comment:
2014?

Reply:
Are these classes just brought over from an existing component? That would explain the 2014 copyright?

Reply:
Correct. Many classes were ported from the existing FAT